core/block_svg.ts

```diff
@@ -1000,6 +1000,8 @@ export class BlockSvg extends Block implements IASTNodeLocationSvg,
       this.comment = null;  // For backwards compatibility.
     }
     if (this.rendered) {
+      // Icons must force an immediate render so that bubbles can be opened
```

Comment:
Can we instead maybe add the comment icon in the `afterQueuedRenders` or whatever we called it? I don't actually know what I'm talking about here so feel free to ignore me if this is nonsense :p

otherwise: nit: (here and below) immediately typo

Reply:
Wrt your comment about `finishQueuedRenders`, that doesn't solve for this use case because the icon needs to be appended to the block before the render is triggered so the renderer can position it correctly.

But now that I'm thinking about this again, I'm going to investigate this one more time.

I was trying to solve for the use case of:
```
myBlock.setCommentText('yay a comment');
myBlock.getCommentIcon().setVisible(true);
```

I thought that for this to correctly position the bubble, the `setCommentText` method must:
  1) Append the comment to the block.
  2) Rerender the block to position the icon.
  3) Tell the icon what its new position is.

So that when `setVisible` is called, the icon has the correct information to position the bubble.

But thinking about it again, if we use the render queueing it should look like:
  1) Append the comment to the block.
  2) Open the bubble at the incorrect position.
  3) Rerender the block to position the icon.
  4) Tell the icon what it new position is.
  5) Update the position of the bubble for the new position of the icon.
  6) Actually paint the frame to the screen.

This should still be correct. But it was causing some unit tests to fail, so I need to remember why that was.